examples/patient-hub/README.md

```diff
@@ -0,0 +1,115 @@
+# Data Hub Many-To-One Example: Integrating Patient Hub 
+
+This example project will demonstrate, how we can use custom steps to integrate data from multiple sources into one.
+
+## Scenario
+
+- Patient Hub project has four entities `Diagnoses`, `Labs`, `Admissions` and `Patients`. Diagnoses and Labs entity needs to be integrated with Admissions and finally Admissions entity needs to be integrated with Patients. This is a many-to-one use case, where we want to integrate data from multiple entities into a final one called `Patients`.
+- After harmonization, `Patients` entity should have 
+    - all the properties from `Patients` entity
+    - startDate, endDate, Labs and Diagnoses fom `Admissions` entity
+
+## Predefined Flows
+
+The project has flows predefined for integrating the patients' data.
+
+- **Diagnoses**: Has a step for ingesting all the patients' diagnosis dataset.
+- **Labs**: Has a step for ingesting all the patients' labs dataset.
+- **Admissions**: Has an ingestion step to ingest all the admissions dataset and a custom step to integrate `Labs` and `Diagnoses` entities with `Admissions`.
+- **Patients**: Has an ingestion step to ingest all the patients' dataset and a custom step to integrate the harmonized `Admissions` entity with `Patients`.
+
+## How to Integrate the Patient Hub
+
+1. Using Data Hub QuickStart with a clean MarkLogic server, select and install the project folder: examples/patient-hub.
+2. Run the `Diagnoses` flow. This ingests Diagnoses documents into the staging database. You can view the documents in Browse Data view for Staging.
+3. Run the `Labs` flow. This ingests Labs documents into the staging database. You can view the documents in Browse Data view for Staging.
+4. Run the `Admissions` flow. It will run two steps, one for ingesting Admissions documents and a custom step to harmonize them with `Diagnoses` and `Labs` entities, into the staging database. The custom step searches and harmonizes the Labs and Diagnoses instances that match a particular AdmissionID and PatientID. You can view the documents in Browse Data view for Staging. The harmonized `Admissions` entity instances start with the URI `/admissionsComplete`.
+5. Run the `Patients` flow. It will run two steps, one for ingesting Patients documents and a custom step to harmonize them with `Admissions` entity(harmonized in the previous step) into the staging database. The custom step searches and harmonizes the Admissions instances that match a particular PatientID. You can view the documents in Browse Data view for Final. The harmonized `Patients` entity instances start with the URI `/patients/admissions`.
```

Comment:
I think this is a typo - the patients are harmonized into the final database, not the staging database.